scripts/ci/pipeline-netcore-runtime.yml

```diff
@@ -73,7 +74,9 @@ stages:
 
         - bash: |
             cd netcore
-            ./build.sh -c $(_BuildConfig)
+            ./build.sh -c $(_BuildConfig) --llvm --rebuild
+            cp ../mono/mini/.libs/libmonosgen-2.0.so ../mono/mini/.libs/libmonosgen-2.0-llvm.so
```

Comment:
This should be a new build step at least but we should probably clone the whole lane to make the execution of the tests easier

Reply:
that was just an experiment to see the CI results, we indeed need a separate lane for llvm